src/components/WalletDetails/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './walletDetails.module.css';
+import UserWelcome from './UserWelcome';
+import { useUserWallet } from './walletDetails.hook';
+import WalletItem from './WalletItem';
+
+const WalletDetails: React.FC = () => {
+  const wallet = useUserWallet();
+  const walletItems = wallet.userWalletDetails.map((item, index) => (
+    <WalletItem key={item.name} currency={item.name} value={item.value} index={index} />
+  ));
+
+  return (
+    <main className={styles.wallet_details_container}>
```

Comment:
why is this in main tag ?

Reply:
it is the main section of the UI so added in in main